web3.md

```diff
@@ -1692,16 +1714,36 @@ Gas Limit Call
     syntax KItem ::= "#firefly_setGasLimit"
  // ---------------------------------------
     rule <k> #firefly_setGasLimit => #rpcResponseSuccess(true) ... </k>
-         <params> [ GLIMIT:String, .JSONs ] </params>
-         <gasLimit> _ => #parseWord( GLIMIT ) </gasLimit>
+         <params>          [ GLIMIT:String, .JSONs ] </params>
+         <gasLimit>        _ => #parseWord( GLIMIT ) </gasLimit>
+         <defaultGasLimit> _ => #parseWord( GLIMIT ) </defaultGasLimit>
```

Comment:
You can instead use just one rule for loading by doing:

```
    <params> [ GLIMIT:String => #parseWord(GLIMIT) , .JSONs ] </params>
```

If that gives you a typing error, you could do:

```
    <params> [ GLIMIT:String, .JSONs ] => [ #parseWord(GLIMIT) , .JSONs ] </params>
```

So that there is only one loader rule (the next rule).